fastlane_bot/helpers/txhelpers.py

```diff
@@ -598,7 +579,7 @@ def build_tx(
                 "from": self.wallet_address,
                 "nonce": nonce,
             }
-        if value is not None:
+        if value > 0:
```

Comment:
Can remove the 'if'

Reply:
Is there a reason we should include the value field when it's not used? Could this cause any unexpected behaviors?

Reply:
Updated